si.tano.Vremenar.yaml

```
@@ -0,0 +1,59 @@
+---
+app-id: si.tano.Vremenar
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vremenar
+rename-icon: vremenar
+rename-desktop-file: vremenar.desktop
+copy-icon: true
+finish-args:
+  # Required due to being a GUI application
+  - "--socket=x11"
```

```suggestion
  - "--socket=fallback-x11"
```

Since you have wayland, this is prefered... but...